### PR TITLE
Phoenix update

### PR DESCRIPTION
**Pull request by kulmann**
This PR updates phoenix to v0.18.0 and adjusts accounts and settings accordingly.

### CODE REVIEW

**Comment by kulmann** [comment]
Updated the phoenix commit id for tests to solve failing webUI CI.

File: accounts/ui/components/accounts/AccountsList.vue [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
         <oc-table-row class="fix-table-header">
-          <oc-table-cell shrink type="head">
+          <oc-table-cell shrink type="head" class="uk-text-center">
             <oc-checkbox
-              :value="areAllAccountsSelected"
-              :label="$gettext('Select all users')"
-              hide-label
-              @change="toggleSelectionAll"
-              class="uk-margin-small-left"
+                class="oc-ml-s"
```
Comment (LukasHirt) [inline accounts/ui/components/accounts/AccountsList.vue:8]: Some odd change of indentation?

**Review by LukasHirt — APPROVED** [review]
One super minor thing. Otherwise all good 👍

File: accounts/ui/components/accounts/AccountsList.vue (hunk 1 above)
Comment (kulmann) [reply accounts/ui/components/accounts/AccountsList.vue:8]: True, that is incorrect indentation. We should setup prettier in accounts and settings UI....

I merged the PR with this. I have another PR touching the accounts and settings UI anyway (moving the navItems to the user menu). I will fix the indentation in that PR then. Thanks for spotting it!

**Pull request merged**